staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/admission.go

```diff
@@ -253,6 +257,18 @@ func (a *MutatingWebhook) Admit(attr admission.Attributes) error {
 	return a.convertor.Convert(versionedAttr.Object, attr.GetObject())
 }
 
+// TODO: factor into a common place along with the validating webhook version.
+func (a *MutatingWebhook) shouldSkipAllHooks(attr admission.Attributes) bool {
```

Comment:
Optional nit: in this case I think it's best to highlight the reason for skipping-- maybe "isWebhookConfigurationResource"?

If we grow additional reasons here, then I'd go for the current name.